src/TouchFreeToolingTypes.ts

```diff
@@ -8,7 +9,7 @@ export class VersionInfo {
 
     // Variable: ApiVersion
     // The current API version of the Tooling.
-    public static readonly ApiVersion: string = '1.5.0';
+    public static readonly ApiVersion: string = '1.6.0';
```

Comment:
👀  TF_TOOLING_API_VERSION in gitlab-ci.yml should also be updated

Reply:
You're right, though I have no idea why that says TOOLING API, the API version is the API version across the whole product? 😕 Any insights?

Reply:
No idea - I know @Craig-J was looking into making there one truth for the API version